doc/PAGES/MeshTutorials/MeshTutorial_07_3DExtrudedMesh.h

```diff
@@ -76,16 +76,16 @@ cylinder),
  ### LV of pre-defined type
  \code
 -- Logical Volumes
-my_LV = LogicalVolumeCreate(RCC, 0, 0, 0.1, 0, 0, 0.2, 0.4)
+my_LV = logvol.Create(RCC, 0, 0, 0.1, 0, 0, 0.2, 0.4)
```

Comment:
I think this is an old routine and that now the logical volumes are hooked up to the input parameters syntax. I would avoid having Lua functions called `Create` outside of those from generated from the factory. I would make this `logvol.CreateLogicalVolume` instead, similar to `aquad.CreateProductQuadrature`. Eventually we should change this to use be `logvol.RCC.Create` or whatever it ends up being.

Reply:
This must be some spurious diff. I find that almost everything in `doc/PAGES` is in some degree out-of-date. I would personally just delete the dir unless there is something valuable there.

But, I agree that the `logvol.CreateLogicalVolume` needs to be replaced by the `logvol.XYZ.Create` API. I think I can look into that in a follow-up PR. The objects registered by `OpenSnRegisterXYZ` macros are not in their proper modules, yet. That will be my next step.